### DISCUSSION TITLE
Define the collection name as a style class inside the page collection

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [directus/directus#7374] [Feature Requests]
We need customize the style only for one collection page, if the collection name as a style class, maybe it's easy customize the collection page.

Make style sometimes it's necessary for special conditions, like spacement, padding and another things. 

![Captura de Tela 2021-08-12 às 10 45 32](https://user-images.githubusercontent.com/379850/129207817-14734d8d-3ed4-4026-88c9-69eb9763e366.png)

**Comment by rijkvanzanten** [comment]
Sounds like we could put the current route full path in a top level class. That should handle all these different cases 🤔

**Reply by benhaynes** [reply]
> current route full path in a top level class

Exploded? I think it would be nice if you could target different scopes (eg: all Collection Detail Pages vs a specific one).

`/admin/collections/candidates` would become:
`collection-detail collection-candidates`

`/admin/collections/candidates/123` would become:
`item-detail collection-candidates item-123`

**Reply by rijkvanzanten** [reply]
> I think it would be nice if you could target different scopes

Sounds like a css problem at that point! You can use the "string contains" selector in CSS to easily target every page that starts with `/collections/candidates` 🤔

**Reply by benhaynes** [reply]
Just seems more complex to target only a collection detail page if you are on an item detail page.

If you're on the `/collections/candidates/123` page, then you'll have to do some fancier stuff to not match a search for `/collections`... right? 

Same applies to target all item-detail pages within a collection, I think.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
